extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -513,6 +513,7 @@ export class AmpSlideScroll extends BaseSlides {
    *     it available for display.
    * @note Element must be laid out.
    * @param {number} newIndex Index of the slide to be displayed.
+   * @return {boolean} true if the slide changed, otherwise false.
```

Comment:
PR Looks good, you should add/update unit tests on this function to check for the boolean values.